src/components/Dropdown/dropdown.module.scss

```diff
@@ -28,7 +28,7 @@
     opacity: 0;
     white-space: normal;
     z-index: $z-index-400;
-    max-height: 34vh;
+    max-height: 50vh;
```

Comment:
If a button sits in the middle of the viewport, 50vh would be slightly too large for full visibility above or below the button.